runner/context.ts

```diff
@@ -18,12 +18,17 @@ import {BrowserRunner} from './browserrunner';
 import * as config from './config';
 import {Plugin} from './plugin';
 
-type Handler = (o: {}, callback: (err: any) => void) => void;
+type Handler = (o: {}) => Promise<any>;
```

Comment:
Good question, `any` is better here. I started with `{}` in a number of places because it would catch and flag assignments so I could track down all properties. This just passes the object through and we don't know anything about it (because we dispatch many messages) so `any` is better.